### PR TITLE
Change statistics implementation to visits by date

### PR DESCRIPTION
**Pull request by Maryam676**
- Class dashboard now displays total student visits by date (extension)
- Original implementation of total visits for corresponding classes now deleted
- Color scheme changed

### CODE REVIEW

File: src/main/webapp/dashboard.js [hunk 1]
```diff
@@ -1,58 +1,72 @@
...
+    var numVisits = visits.classVisits;
+
+    var tempDataHolder = []; // To be pushed into datatable after updating
+
+    // Loop through both lists and add info sets for each class 
+    for (var i = 0; i < dates.length; i++) {
+      var temp = [];
+      temp.push(dates[i]);
+      temp.push(numVisits[i]);
+      tempDataHolder.push(temp);
```
Comment (nmittu) [inline src/main/webapp/dashboard.js:25]: Do you think in-lining this list will make it more clear `tempDataHolder.push([dates[i], numVisits[i]]);`?

File: src/main/java/com/google/sps/servlets/VisitsByDate.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    try {
+      authInstance = FirebaseAuth.getInstance(FirebaseAppManager.getApp());
+    } catch (IOException e) {
+      throw new ServletException(e);
+    }
+  }
+
+  // Obtain a query of class visits, filter by unique class, and store the visits by date
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
[the remark this answers is not in the document]
Comment (Maryam676) [reply src/main/java/com/google/sps/servlets/VisitsByDate.java:46]: Marked for extension

File: src/main/java/com/google/sps/servlets/VisitsByDate.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import java.util.Date;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// Once class owner submits a TA email, retrieve that user and
+// add them as a TA to the class
```
Comment (mdchu) [inline src/main/java/com/google/sps/servlets/VisitsByDate.java:28]: is this comment correct?

Comment (Maryam676) [reply src/main/java/com/google/sps/servlets/VisitsByDate.java]: Fixed

**Pull request merged**